src/services/emailDelivery.ts

```diff
@@ -0,0 +1,32 @@
+import { WithValidationError } from "../types/withValidationError"
+import { ApiCallOptions, handleValidationError, postData } from "../base"
+
+export const supportCases = async ({
```

Comment:
Let's go with:

```suggestion
export const sendSupportCase = async ({
```

1. we only send one case, singular reflects that intention
2. let's add `send` prefix to reflect that we're sending some data and be consistent with other naming